packages/big-design/src/components/Datepicker/Datepicker.tsx

```diff
@@ -0,0 +1,91 @@
+import { enAU, enCA, enGB, enUS } from 'date-fns/locale';
+import React, { forwardRef, memo, Ref, useEffect, useState } from 'react';
+import ReactDatePicker, { registerLocale } from 'react-datepicker';
+
+registerLocale('en-US', enUS);
```

Comment:
After thinking about this a bit, instead of manually setting these inside BigDesign, what do you think about exposing an API to register them outside of the component?

My reasoning behind this is so that we can drop the `date-fns` package - one less dependency. Plus I think it should be the application that determines the locales it needs, not BigDesign.